### ISSUE TITLE
Needs installation instructions.

### ISSUE THREAD

**Issue by singingfish** [HaraldJoerg/cperl-mode#15]
I'm afraid I'm too reliant on `M-x package-list-packages` and don't have the chops/tuits to remember how to do it manually.

But I can follow install instructions and I will use this almost daily if someone writes them.

**Comment by HaraldJoerg** [comment]
Fair enough!  I have added a section with instructions to README.md.

Also: Thanks for the feedback!  I'd like to ask a question: Are you using _this_ cperl-mode.el because it supports syntax extensions (and which ones) or because it has bug fixes?  The reason is that the implementation of syntax extensions turned out to be somewhat of a dead end and needs to be re-worked.  It will look differently when it goes into official Emacs (and it will take time).

**Comment by singingfish** [comment]
The main reason I wanted to look at giving it a try was I was using
Syntax::Keyword::Try and I couldn't see a way of fixing the default cperl
indentation.

**Comment by HaraldJoerg** [comment]
Oh, indentation is indeed ... messy.  This is still work in progress.

I did some tests with Syntax::Keyword::Try and found that "vanilla" cperl-mode.el in Emacs 26 and above gets it right for `try` and `catch`, but fails for `finally` -- it treats the line after the `finally` block as a continuation.  Is that the bug you're talking about?

...and then I found that cperl-mode.el from this repository gets it right for `try` and `catch`, but fails for `finally`.  **Ouch.**  I'll push a fix later.
